examples/by_feature/early_stopping.py

```diff
@@ -0,0 +1,246 @@
+# coding=utf-8
```

Comment:
AFAICT, this whole script is almost identical to `examples/nlp_example.py`, just with the addition of early stopping. I wonder: Would it make more sense to add the early stopping code to that script and enable/disable it via an additional command line argument?

Reply:
These scripts are designed to be just that :)

Reply:
Sorry, I do not understand your comment, could you please elaborate? :)

If you mean that the script should stay simple: the `examples/nlp_example.py` already has a couple of command line arguments, so adding one more wouldn't be very costly.

Reply:
The `by_feature` script Re meant to be simple additions to `nlp_example.py` showing just a single feature being added to them, so users aren't overwhelmed with a million lines trying to find and do one thing. The `nlp_example.py` remains core, and small additions as we add new features get added in as feature scripts

Reply:
Ah I see, that makes sense.